cypress/e2e/requests.cy.js

```diff
@@ -1,26 +1,53 @@
 describe('Viewing all requests', () => {
-  it('does not show a request list if the user is logged out.', () => {
-		// Visit a protected route in order to allow cypress to set the cookie and mock the login
-		cy.visit("/requests")
-
-		cy.get('div.alert-heading').contains('Unauthorized').then(() => {
-			cy.log("A logged out user is not able to view requests.")
-		})
-	})
+  describe('as a logged out user', () => {
+    it('should show an error message.', () => {
+      // Visit a protected route in order to allow cypress to set the cookie and mock the login
+      cy.visit("/requests")
+      cy.get('div.alert-heading').contains('Unauthorized').then(() => {
+        cy.log("A logged out user is not able to view requests.")
+      })
+    })
+  })
   
-	it("shows the user's request list if they are logged in.", () => {
-		// Call the custom cypress command to log in
-		cy.login(Cypress.env('TEST_SCIENTIST_USER'), Cypress.env('TEST_SCIENTIST_PW'))
+  describe('as a logged in user', () => {
+    let scientistApiBaseURL = `https://${Cypress.env('NEXT_PUBLIC_PROVIDER_NAME')}.scientist.com/api/v2`
+    let requestList
+
+    beforeEach(() => {
+      // Call the custom cypress command to log in
+      cy.login(Cypress.env('TEST_SCIENTIST_USER'), Cypress.env('TEST_SCIENTIST_PW'))
```

Comment:
could this line be in a `before` block above this `beforeEach` so that the user is signed in once and then stays that way?

Reply:
let me see if that works

Reply:
didn't work. the slate is wiped clean between each test, and since before only runs once it only applied to the test immediately after:
https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Hooks

Reply:
according to this [SO post](https://stackoverflow.com/a/60484658/8079848), it looks like the `before` would need to be placed above the `describe` block in order to work the way I was thinking of. ehh.

I'll leave the decision to you.

Reply:
hmmmm. i tried that, and now they all fail, included the immediately following test 🧐

i think it may have to do with the fact that the thing being called in the before is cy.session. but not sure